proxy/src/main/java/com/wavefront/agent/listeners/tracing/ZipkinPortUnificationHandler.java

```diff
@@ -396,6 +399,16 @@ private void processZipkinSpan(zipkin2.Span zipkinSpan) {
     }
   }
 
+  private boolean sample(Span wavefrontSpan) {
+    if (sampler != null && sampler.sample(wavefrontSpan.getName(),
```

Comment:
if not marked Nullable, dont do null check
also if you do null check then default to true and not false